bot/exts/info/patreon.py

```diff
@@ -0,0 +1,160 @@
+
+        await sending_channel.send(
+            embed=discord.Embed(
+                description=message,
+                colour=colour
+            )
+        )
+
+    async def send_current_supporters(self, channel: discord.TextChannel) -> None:
+        """Send the current list of patreon supporters, sorted by tier level."""
+        await self.bot.wait_until_guild_available()
+
+        guild = self.bot.get_guild(constants.Guild.id)
+
+        tier_1_or_above_patrons = set(guild.get_role(constants.Roles.patreon_tier_1).members)
+        tier_2_or_above_patrons = set(guild.get_role(constants.Roles.patreon_tier_2).members)
+        tier_3_patrons = set(guild.get_role(constants.Roles.patreon_tier_3).members)
+
+        tier_1_patrons = tier_1_or_above_patrons - tier_2_or_above_patrons - tier_3_patrons
+        tier_2_patrons = tier_2_or_above_patrons - tier_3_patrons
+
+        tier_1_patrons = {f"{patron.mention} ({patron.name}#{patron.discriminator})" for patron in tier_1_patrons}
+        tier_2_patrons = {f"{patron.mention} ({patron.name}#{patron.discriminator})" for patron in tier_2_patrons}
+        tier_3_patrons = {f"{patron.mention} ({patron.name}#{patron.discriminator})" for patron in tier_3_patrons}
+
+        embed_list = []
+
+        embed_patron_info = discord.Embed(
+            title="Patreon Supporters",
+            description=(
+                "Here is a full list of this months Python Discord patrons!\n\nWe use the money from Patreon to offer "
+                "excellent prizes for all of our events. Prizes like t-shirts, stickers, microcontrollers that support "
+                "CircuitPython, or maybe even a mechanical keyboard.\n\nYou can read more about how Patreon supports "
+                "us, or even support us yourself, on our Patreon page [here](https://pydis.com/patreon)!"
+            )
+        )
+
+        embed_list.append(embed_patron_info)
+
+        if tier_1_patrons:
+            embed_tier_1 = discord.Embed(
+                title="Tier 1 patrons",
+                description="\n".join(tier_1_patrons),
+                colour=guild.get_role(constants.Roles.patreon_tier_1).colour
+            )
+            embed_list.append(embed_tier_1)
+
+        if tier_2_patrons:
+            embed_tier_2 = discord.Embed(
+                title="Tier 2 patrons",
+                description="\n".join(tier_2_patrons),
+                colour=guild.get_role(constants.Roles.patreon_tier_2).colour
+            )
+            embed_list.append(embed_tier_2)
+
+        if tier_3_patrons:
+            embed_tier_3 = discord.Embed(
+                title="Tier 3 patrons",
+                description="\n".join(tier_3_patrons),
+                colour=guild.get_role(constants.Roles.patreon_tier_3).colour
+            )
+            embed_list.append(embed_tier_3)
+
+        await channel.send(embeds=embed_list)
+
+    @commands.command("patrons")
```

Comment:
There's no check on this, anybody can send this anywhere.

Reply:
How would I let admins (or whatever role we want) send it in community meta, while the rest of the community can only send in the bot command channels?

Reply:
I have limited the command to list patrons to `bot_commands`